caluma/caluma_logging/middleware.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+from caluma.caluma_logging.models import AccessLog
+
+
+class AccessLogMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
+        # user is available only after the request was processed
+        body = json.loads(request.body.decode("utf-8"))
+
+        AccessLog.objects.create(
```

Comment:
Maybe we should try to parse the request just a bit further: Figure out if it's a mutation or not, so we can easily search for the type of request. Might even go as far and store the query / mutation name as well if it's not too much work

Reply:
I agree this could be usefull information but then I would rather move this somewhere deeper after the graphql parsing is done, not doing it ourselfs again...
Maybe the graphene MIDDLEWARE setting?

Reply:
As seen above, we just do the parsing again. Not optimal from a performance standpoint but shouldn't be too bad either.